File: testprojects/java/benchmark/infrastructure/database/objectdb/ObjectdbConnectionSettings.java
package infrastructure.database.objectdb;

import infrastructure.database.DataCollection;

import java.sql.Connection;
//Functional requirement 2.6
//Test case 28: All classes in package infrastructure.database.objectdb must extend class infrastructure.database.DatabaseCollection
//Result: FALSE
public abstract class ObjectdbConnectionSettings extends DataCollection {
	public Connection getConnection() {
		return null;
	}
}

File: testprojects/csharp/benchmark/infrastructure/database/objectdb/ObjectdbConnectionSettings.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Data.SqlClient;

namespace CSharpBenchmark.infrastructure.database.objectdb
{
    //Functional requirement 2.6
    //Test case 28: All classes in package infrastructure.database.objectdb must extend class infrastructure.database.DatabaseCollection
    //Result: FALSE
    public abstract class ObjectdbConnectionSettings : DataCollection
    {
        public SqlConnection getConnection()
        {
            return null;
        }
    }
}